packages/api/pages/api/languages/[code]/verses/[verseId]/suggestions.ts

```diff
@@ -91,7 +89,7 @@ export default createRoute<{ code: string; verseId: string }>()
       const words = await client.$queryRaw<WordQuery[]>`
         SELECT
           w.id AS "wordId",
-          target.gloss AS "gloss", target.state AS "state",
+          target.gloss AS "gloss",
```

Comment:
It doesn't look like we use the `gloss` column or the `target` subquery anywhere.

Reply:
We are using the target gloss in `generateMachineGlossesForVerse` to determine if we need to generate a machine gloss or not. If there is already a gloss, then we don't need to bother spending money to use the google translate api to generate a machine gloss